MILS/Functions/es_main.m

```diff
@@ -4,7 +4,7 @@
 %% Code
     if algo == "Default Block"        
         % Check this!!
-        q = eul2quat([deg2rad(sis_output.attitude(1)),deg2rad(-sis_output.attitude(2)),deg2rad(-sis_output.attitude(3))],'ZYX');
```

Comment:
Add exact formulae here and in the README so that errors like this don't occurr.